glossary.asciidoc

```diff
@@ -362,45 +357,45 @@ peer::
     In particular, they are connected via an encrypted, authenticated communication over a TCP Socket.
 
 penalty transaction::
-    Look at the Breach Remedy Transaction.
+    See Breach Remedy Transaction.
 
 private channel:
     A channel not announced to the rest of the network. 
     Technically "private" is a misnomer as these channels can still be identified through routing hints and commitment transactions.
-    We suggest they are better described as "unannounced" channels.
+    They are better described as "unannounced" channels.
     With an unannounced channel, the channel partners can send and receive payments between each other as normal.
-    However the rest of the network will not be aware of it and so cannot typically use it to route payments.
-    As the number of and capacity of unanounced channels is unknown, the total public channel count and capacity only accounts for a portion of the total Lightning Network.
+    However, the rest of the network will not be aware of the channel and so cannot typically use it to route payments.
```

Comment:
cannot is actually not accurate. Better: 

so it will typically not be used by other nodes to route payments. (of course it can do it and it also might happen as some people might be aware of the `unannounced` channel)

Reply:
Ok.  Agreed.